web/skins/classic/views/js/watch.js

```diff
@@ -1190,7 +1190,33 @@ function cycleNext() {
   }
   clearInterval(intervalId);
   monitorStream.kill();
-  window.location.replace('?view=watch&cycle='+cycle+'&mid='+monitorData[monIdx].id+'&mode='+mode);
+
+  // +++ Старт следующего монитора
+  monitorStream = new MonitorStream(monitorData[monIdx]);
+  const img = document.getElementById('liveStream'+monitorData[monIdx-1].id);
+  const src = img.src;
+  if (src) {
+    const url = new URL(src);
+    url.searchParams.set('monitor', monitorData[monIdx].id);
+    url.searchParams.delete('connkey');
+    url.searchParams.set('mode', 'single');
```

Comment:
why start with single?  Let's go straight to streaming

Reply:
doesn't really matter I suppose

Reply:
Today the decision was made to do everything completely differently.

Reply:
It was “such a small thing” compared to the current code changes :)